build.sh

```diff
@@ -264,8 +264,8 @@ do
         GROUP_ID=50
       fi
       {
-        cat share/docker/Dockerfile
-        grep -vF 'FROM avro-build-ci' share/docker/DockerfileLocal
+        sed -r "s/openjdk:8/openjdk:${JAVA:-8}/g" share/docker/Dockerfile
```

Comment:
I've introduced this, but this is isn't the proper way of doing this. Sorry for that. I think we should use Docker `ARG`'s, similar to Airflow: https://github.com/apache/airflow/blob/master/Dockerfile#L18-L19

Reply:
Wuff. It is a stretch to use ARG for defining the base image. It invalidates the cache entirely, so it isn't materially better than having distinct Dockerfiles. For my two cents, we should consider using a base image like ubuntu, and then installing both jdks in it and switching jdks internally on demand.

Reply:
Ah, I assumed that the `sed` was to work around Yetus.  If I understood correctly, `--build-arg` support is coming in the next Yetus release.

Currently we're using an "ubertools" container for local dev *and* remote CI, but it's not particularly straightforward or easy to extend.  

I can give the two JDKs strategy a try and see if it much difference.